tool/src/org/antlr/v4/gui/PostScriptDocument.java

```diff
@@ -84,7 +84,7 @@ public String getPS() {
 	public void boundingBox(int w, int h) {
 		boundingBoxWidth = w;
 		boundingBoxHeight = h;
-		boundingBox = String.format(Locale.US, "%%%%BoundingBox: %d %d %d %d\n", 0,0,
+		boundingBox = String.format(Locale.US, "%%%%BoundingBox: %d %d %d %d %n", 0,0,
```

Comment:
The PostScript files are unlikely to be read as text. According to the language specification, LF and CRLF are treated the same way, so the changes in this file don't seem to help or hurt.